inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -93,18 +93,19 @@ class ConstantWriter {
             m_binary_output.write(ptr, size);
             return offset;
         }
-        // The biggest supported models have at maximum 1-2 thousand constant nodes,
-        // with 64 bit hash that gives a probability around 1 in 10 trillion that a
-        // hash collision will appear. Because of this, a choice has been made to
-        // not perform collision detection and keep the hashing quick and seamless.
+        // This hash is weak (but efficient) and must be replace with some other
+        // more stable hash algorithm. For example current hash algorithms gives
```

Comment:
```suggestion
        // more stable hash algorithm. For example current hash algorithm gives
```

Reply:
If you don't mind, I'll fix this in next PR.